projects/schematics/src/migrations/3_0/storefinder/storefinder_spec.ts

```diff
@@ -46,7 +46,7 @@ import { StoreFinderModule } from '@spartacus/misc/storefinder';
     B2cStorefrontModule.withConfig({}),
     StoreFinderRootModule,
     StoreFinderModule,
-    
+
```

Comment:
Do we need this empty line? :)

Reply:
yes, but it has to be empty with spaces unfortunately...